### PR TITLE
Project Music Releases - Laima & Matilda

### PR DESCRIPTION
**Pull request by leelalaim**
Project was created by Laima and Matilda (Lions team)

Link to my deployed project:
https://music-releases-site.netlify.app/

My reflections on how this week's project turned out:
This weeks project went good. The problems that we faced (regarding adding a commas and and symbols between different artists, aligning, different view on Safari browser..) were sorted with the help of StackOverflow and code coach.

Things I'd like to have clarified or explained in more detail:
- All of our component are now in the Album.js and then we import the Album component to App.js. is that a correct way of structuring the components or they all should have been in App.js instead of an Album?

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
+              artist={album.artists.map((item, index) => <a>{item.name}</a>)}
+              hrefArtist={album.artists.map((item) => item.external_urls.spotify)}
+              key={album.id}/>
+             )
+          })
+        }
+      </div>
     </div>
+    </>
   )
```
Comment (KarolCislowski) [inline code/src/App.js:35]: If your component returns only one root element, there is no need to wrap it around React Fragment.

In this case WE NEED to use React Fragment :

```jsx
return (
	<div>My element 1</div>
        <div>My element 2</div>
);
```

so, after necessary changes it looks like this :

```jsx
return (
	<>
          <div>My element 1</div>
	  <div>My element 2</div>
	</>
);
```

But in this case WE DON'T NEED to use React Fragment :

```jsx
return (
	<div>My element 1</div>
);
```

File: code/src/components/Icons.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const Icons = (prop) => {
```
Comment (KarolCislowski) [inline code/src/components/Icons.js:3]: You do not use any props so you do not need to specify it here.

File: code/src/components/AlbumTitle.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const AlbumTitle = (prop) => {
```
Comment (KarolCislowski) [inline code/src/components/AlbumTitle.js:3]: If you destructure your props like this:

```jsx
export const AlbumCard = props => {
  const { title, image } = props
}
```

or like this:

```jsx
export const AlbumCard = ({ title, image }) => {
  //
}
```

**Review by KarolCislowski — APPROVED** [review]
It is a very nice page 🦄
I left a few tips below to make it even better. 🦄

**Comment by leelalaim** [comment]
Requested changes were made

**Pull request closed without merging**